discretisedfield/region.py

```diff
@@ -672,9 +830,19 @@ def allclose(self, other):
 
         """
         if isinstance(other, self.__class__):
-            return np.allclose(self.pmin, other.pmin, atol=0) and np.allclose(
-                self.pmax, other.pmax, atol=0
-            )
+            if atol is None:
+                atol = np.min(self.edges) * self.tolerance_factor
+            elif not isinstance(atol, numbers.Number):
+                raise TypeError(f"{type(atol)=} is not a number.")
+
+            if rtol is None:
+                rtol = np.min(self.edges) * self.tolerance_factor
+            elif not isinstance(rtol, numbers.Number):
+                raise TypeError(f"{type(rtol)=} is not a number.")
```

Comment:
This looks good, we have to extend the documentation.

Reply:
@lang-m what are you thoughts on this being `isclose` rather than `allclose`?

Reply:
Do you mean adding an `isclose` method? Or renaming this method?

Reply:
Renaming this method. We are checking if one region is close to another region (similar to `np.isclose`). Whereas we would use `allclose` in field as it checks all of the values (similar to `np.allclose`)

Reply:
But we check pmin and pmax of the region and return a single bool. For `isclose` we could also expect individual boolean values for pmin and pmax. I think I would stick to `allclose`.